glossary.yml

```
@@ -379,7 +384,12 @@
       Um método de desenvolvimento de software que enfatiza vários passos pequenos e
       feedback contínuo ao invés de planejamento de longo prazo. [Programação exploratória](#exploratory_programming)
       costuma ser ágil.
-
+ es:
+    term: "desenvolvimiento ágil"
+    def: >
```

```suggestion
  es:
    term: "desenvolvimiento ágil"
    def: >
```

Fixing indentation errors.